plugins/modules/database/misc/odbc.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, John Westcott <john.westcott.iv@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: odbc
+author: "John Westcott IV (@john-westcott-iv)"
+version_added: "1.0.0"
+short_description: Execute SQL via ODBC
+description:
+    - Read/Write info via ODBC drivers.
+options:
+    dsn:
+      description:
+        - The connection string passed into ODBC
+      required: true
```

Comment:
```suggestion
      description:
        - The connection string passed into ODBC.
      required: yes
```
according to the doc

Reply:
Do you have a reference to the doc? If I look at https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-fields I don't see `yes` (not that we can't use yes as a bool) or looking at the example on https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_general.html#starting-a-new-module they have `required: true`.

Reply:
https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#examples-block
```If your examples use boolean options, use yes/no values. Since the documentation generates boolean values as yes/no, having the examples use these values as well makes the module documentation more consistent.```
Anyway **it's just a recomendation** from me and both ways are acceptable, **though it'll be converted to `yes` / `no`.**

@acozine @samccann what do you think?
Maybe makes sense to change `true/false` -> `yes/no` everywhere in the doc examples?